updating/updating-restricted-network-cluster/restricted-network-update-OSUS.adoc

```diff
@@ -0,0 +1,136 @@
+:_content-type: ASSEMBLY
+[id="updating-restricted-network-cluster-OSUS"]
+= Updating a restricted network cluster using the OpenShift Update Service
+include::_attributes/common-attributes.adoc[]
+:context: updating-restricted-network-cluster-OSUS
+
+toc::[]
+
+[id="update-restricted-network-cluster-update-service"]
+
+include::modules/update-service-overview.adoc[leveloffset=+1]
+
+.Additional resources
+
+* xref:../../updating/understanding-upgrade-channels-release.adoc#understanding-upgrade-channels_understanding-upgrade-channels-releases[Understanding upgrade channels and releases]
+
+For clusters with internet accessibility, Red Hat provides over-the-air updates through an {product-title} update service as a hosted service located behind public APIs. However, clusters in a restricted network have no way to access public APIs for update information.
+
+To provide a similar update experience in a restricted network, you can install and configure the OpenShift Update Service locally so that it is available within a disconnected environment.
+
+The following sections describe how to provide over-the-air updates for your disconnected cluster and its underlying operating system.
```

Comment:
nit: I realize "over-the-air" is part of the internal updates-team name, but I'm personally not clear on what it is adding.  Yes, admins can update OpenShift clusters without needing a stack of floppy disks or equivalent.  But that's true of basically all software these days, so possibly drop "over-the-air" throughout these docs?  Or am I missing what "over-the-air" is trying to convey?

Reply:
remove mention of over-the-air per Trevor's comment here

Reply:
I'll remove these mentions now